docs (release-highlights): Adds 6.6 highlights (#5045)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/releases/release-highlights.md

    - [@patternfly/react-log-viewer@6.5.0](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/6.5.0) ([changelog](https://github.com/patternfly/react-log-viewer/releases))
    - [@patternfly/react-topology@6.6.0](https://www.npmjs.com/package/@patternfly/react-topology/v/6.6.0) ([changelog](https://github.com/patternfly/react-topology/releases))
    - [@patternfly/react-user-feedback@6.4.0](https://www.npmjs.com/package/@patternfly/react-user-feedback/v/6.4.0)
    - [@patternfly/react-virtualized-extension@6.2.0](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/6.2.0) ([changelog](https://github.com/patternfly/react-virtualized-extension/releases))

### Component enhancements

This release brings targeted improvements to the following components:

- **Avatar:** You can now add icons and use color variants to create more expressive user representations.
- **Banner:** Added pill variant.
- **Navigation:** Expandable navigation items now support an optional icon prop that displays before the title.

> - **Table:** We fixed a critical Safari-specific layout bug in tables with sticky headers.

Reviewer: I believe it was specifically when a table had sticky and nested headers. cc @mcoker Am I recalling correctly or did the bug apply to sticky headers in general?

Reviewer: Michael confirmed all sticky headers so lgtm!

import '../get-started/get-started.css';
import { Divider, Timestamp } from '@patternfly/react-core'; 

<Timestamp date={new Date(2026, 5)}>June 2026</Timestamp>

## PatternFly 6.6

This release refines core components and improves visual consistency to help you build better experiences. You'll find focused updates to typography, icons, and foundational components, plus important cross-browser compatibility fixes.

> Key highlights include updated Red Hat font integration with refined font-weight tokens, new avatar component capabilities with icon and color variant support, table footer element support, and Safari compatibility improvements. This release focuses on polish&mdash;refining interaction states, fixing visual inconsistencies, and enhancing component flexibility across the design system.

Reviewer: Can we remove the mdash here and a bit further below?

Also, the tfoot change Michael was requesting be removed is also mentioned in this paragraph and should be removed too.

    - [@patternfly/react-data-view@6.5.0-prerelease.2](https://www.npmjs.com/package/@patternfly/react-data-view/v/6.5.0-prerelease.2) ([changelog](https://github.com/patternfly/react-data-view/releases))
    - [@patternfly/react-log-viewer@6.4.0-prerelease.2](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/6.4.0-prerelease.2) ([changelog](https://github.com/patternfly/react-log-viewer/releases))
    - [@patternfly/react-topology@6.6.0-prerelease.1](https://www.npmjs.com/package/@patternfly/react-topology/v/6.6.0-prerelease.1) ([changelog](https://github.com/patternfly/react-topology/releases))
    - [@patternfly/react-user-feedback@6.4.0-prerelease.2](https://www.npmjs.com/package/@patternfly/react-user-feedback/v/6.4.0-prerelease.2)
    - [@patternfly/react-virtualized-extension@6.2.0](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/6.2.0) ([changelog](https://github.com/patternfly/react-virtualized-extension/releases))

### Component enhancements

> This release brings targeted improvements to core components:

Reviewer: This may just be me, but with "core" I immediately thought of the core/CSS package and not the "core of PatternFly" which is what I think it was going for. What do others think? I'm probably fine leaving it or rephrasing.

Reviewer 2: +1, it's probably fine but I think a different word would be better. Maybe "_This release brings targeted improvements to the following components:_"?

- **Avatar:** You can now add icons and use color variants to create more expressive user representations.
- **Table:** Added support for table footer elements using the tfoot element, giving you more flexibility for data table layouts. We also fixed Safari-specific rendering issues with table header backgrounds.
- **Buttons:** Refined clicked states for inline link buttons to improve visual feedback.

> - **Icons:** Updated caret icons across dual list selectors, pagination controls, and toolbar components for improved visual consistency.

Reviewer: More icons updates than the caret icons went in, but I'm not sure if it's worth calling them out individually or just make a general note about the in-progress refactor of the PF icons to the RH icons.

### Typography and design tokens

We've updated Red Hat font integration with refined font-weight token values. These updates ensure better alignment with Red Hat Design System standards while maintaining consistency across all PatternFly themes introduced in 6.5&mdash;including Project Felt, glass mode, and high-contrast mode.

### Visual polish

This release includes visual refinements across multiple components:
- Improved alert icon alignment for multiline titles
- Fixed banner link colors
- Corrected borders in panel and docked navigation components
- Refined drag-and-drop ghost row border-radius values

> 

Reviewer 3: Could we add a section for Chatbot here? Maybe somewthing along the lines of:

```
### Alignment on Chatbot AI standards

Our Chatbot extension has made updates to help provide consistent alignment regarding AI standards:

- **MessageBar:** Removed gradient borders and animations for when an AI is "thinking".
- **ChatbotToggle:** Updated the icons for the Chatbot toggle, using the new recommended robot icon when the toggle is collapsed, and the Red Hat brand icon when the toggle is expanded.
- **Message avatars:** Updated the avatar for bot messages to use the new recommended robot icon by default. Customization of user avatars has also been expanded, allowing more custom svgs or initials, following the updates made to the [avatar component[(/components/avatar).
- **ChatbotFootnote:** The recommended verbiage has been updated for more consistency.

In order to take advantage of this 6.7 Chatbot release, you will need to update any other PatternFly packages to their latest 6.6 release versions.
```

Author: Done! Thanks.

import '../get-started/get-started.css';
import { Divider, Timestamp } from '@patternfly/react-core'; 

<Timestamp date={new Date(2026, 5)}>June 2026</Timestamp>

## PatternFly 6.6

This release refines core components and improves visual consistency to help you build better experiences. You'll find focused updates to typography, icons, and foundational components, plus important cross-browser compatibility fixes.

> Key highlights include updated Red Hat font integration with refined font-weight tokens, new avatar component capabilities with icon and color variant support, table footer element support, and Safari compatibility improvements. This release focuses on polish — refining interaction states, fixing visual inconsistencies, and enhancing component flexibility across the design system.

Reviewer 2: * I would take out the table footer comment - that hasn't landed in react yet.
* "Safari compatibility improvements" was fixing a critical table bug in the table in Safari. Specifically, tables with sticky headers were effectively unusable. I would probably describe it as fixing a bug over calling it a "compatibility improvement," but that's just my opinion. Maybe this is better.
* Also the use of an em-dash to interrupt a thought, immediately followed by a list of three with an oxford comma screams AI 😅 But it's also how people write, so there's that!

```suggestion
Key highlights include updated Red Hat font integration with refined font-weight tokens, new avatar component capabilities with icon and color variant support, and Safari compatibility improvements. This release focuses on polish — refining interaction states, fixing visual inconsistencies, and enhancing component flexibility across the design system.
```

  - [@patternfly/react-data-view@6.5.0-prerelease.2](https://www.npmjs.com/package/@patternfly/react-data-view/v/6.5.0-prerelease.2) ([changelog](https://github.com/patternfly/react-data-view/releases))
  - [@patternfly/react-log-viewer@6.4.0-prerelease.2](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/6.4.0-prerelease.2) ([changelog](https://github.com/patternfly/react-log-viewer/releases))
  - [@patternfly/react-topology@6.6.0-prerelease.1](https://www.npmjs.com/package/@patternfly/react-topology/v/6.6.0-prerelease.1) ([changelog](https://github.com/patternfly/react-topology/releases))
  - [@patternfly/react-user-feedback@6.4.0-prerelease.2](https://www.npmjs.com/package/@patternfly/react-user-feedback/v/6.4.0-prerelease.2)
  - [@patternfly/react-virtualized-extension@6.2.0](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/6.2.0) ([changelog](https://github.com/patternfly/react-virtualized-extension/releases))

### Component enhancements

This release brings targeted improvements to core components:

- **Avatar:** You can now add icons and use color variants to create more expressive user representations.

> - **Table:** Added support for table footer elements using `<tfoot>`, giving you more flexibility for data table layouts. We also fixed Safari-specific rendering issues with table header backgrounds.

Reviewer 2: Remove the `tfoot` part. Reworded the safari bug but you don't have to use my suggestion. I do think it's important to mention that the bug was with tables w/ sticky headers specifically.

```suggestion
- **Table:** We fixed a Safari-specific layout bug in tables with sticky headers.
```

> - **Buttons:** Refined clicked states for inline link buttons to improve visual feedback.

Reviewer 2: This was also fixing a bug. IMO I don't know if it's worth calling out. If you go to the[ button example page](https://www.patternfly.org/components/button) and hold your mouse down on an inline link button, the underline disappears. We fixed that so it doesn't disappear.

- **Icons:** Updated caret icons across dual list selectors, pagination controls, and toolbar components for improved visual consistency.

### Typography and design tokens

> We've updated Red Hat font integration with refined font-weight token values. These updates ensure better alignment with Red Hat Design System standards while maintaining consistency across all PatternFly themes introduced in 6.5 — including Project Felt, glass mode, and high-contrast mode.

Reviewer 2: Another way of saying all of this is that we
1. Updated the Red Hat fonts in patternfly (Red Hat Text, Red Hat Display, and Red Hat Mono) to the latest versions.
  * Here is the upstream RH Font repo pull request that we pulled in - https://github.com/RedHatOfficial/RedHatFont/pull/75. 
2. Updated the numerical font-weights used for bold body text, and regular and bold heading text.

It might be worth mentioning that if apps have incorrect font-weights in their UIs after this update, to ensure they're using our CSS variables to control the font-weight of text in their app. That will ensure they're always getting the correct numerical font-weight patternfly intends. The variables to use are
* `--pf-t--global--font--weight--body--default`
* `--pf-t--global--font--weight--body--bold`
* `--pf-t--global--font--weight--heading--default`
* `--pf-t--global--font--weight--heading--bold`
 
More info can be found by searching for those tokens in our [all design tokens table](https://www.patternfly.org/foundations-and-styles/design-tokens/all-design-tokens/).

### Visual polish

This release includes visual refinements across multiple components:
- Improved alert icon alignment for multiline titles
- Fixed banner link colors

> - Corrected borders in panel and docked navigation components

Reviewer 2: For the panel, not sure if it's worth mentioning but that just added a border to panels in high-contrast theme.

### Component enhancements

This release brings targeted improvements to the following components:

- **Avatar:** You can now add icons and use color variants to create more expressive user representations.
- **Banner:** Added pill variant for a more compact appearance.
- **Navigation:** NavExpandable now supports icon integration.
- **Table:** We fixed a critical Safari-specific layout bug in tables with sticky headers.
- **Drawer:** Fixed resizable drawer width jump on drag operations.
- **Icons:** Updated icons across the design system for better alignment with Red Hat brand standards, including caret icons in dual list selectors, pagination controls, and toolbar components.

### Typography and design tokens

We've updated Red Hat font integration and font-weight token values:

- Updated Red Hat fonts (Red Hat Text, Red Hat Display, and Red Hat Mono) to the latest versions from the [upstream Red Hat Font repository](https://github.com/RedHatOfficial/RedHatFont/pull/75).

> - Updated the numerical font-weights used for bold body text, and regular and bold heading text.

Reviewer 2: I know I wrote this 😅 but just a small tweak 

```suggestion
- Updated the numerical font-weights used for bold body text, as well as regular and bold heading text.
```

**Important:** If your app displays incorrect font-weights after this update, ensure you're using our CSS variables to control font-weight. This ensures your app always uses the correct numerical font-weight PatternFly intends. The variables to use are:
- `--pf-t--global--font--weight--body--default`
- `--pf-t--global--font--weight--body--bold`
- `--pf-t--global--font--weight--heading--default`
- `--pf-t--global--font--weight--heading--bold`

More information can be found by searching for those tokens in our [all design tokens table](https://www.patternfly.org/foundations-and-styles/design-tokens/all-design-tokens/).

### Alignment on Chatbot AI standards

Our Chatbot extension has made updates to help provide consistent alignment regarding AI standards:

- **MessageBar:** Removed gradient borders and animations for when an AI is "thinking".
- **ChatbotToggle:** Updated the icons for the chatbot toggle, using the new recommended robot icon when the toggle is collapsed, and the Red Hat brand icon when the toggle is expanded.

> - **Message avatars:** Updated the avatar for bot messages to use the new recommended robot icon by default. Customization of user avatars has also been expanded, allowing more custom svgs or initials, following the updates made to the [avatar component](/components/avatar).

Reviewer 3: Just a slight tweak based on convo from office hours:

```suggestion
- **Message avatars:** Updated the avatar for bot messages to use the new recommended robot icon, which will no longer be able to be overridden in favor of consistency across products. Customization of user avatars has also been expanded, allowing more custom svgs/icons or initials, following the updates made to the [avatar component](/components/avatar).
```

### Component enhancements

This release brings targeted improvements to the following components:

- **Avatar:** You can now add icons and use color variants to create more expressive user representations.

> - **Banner:** Added pill variant for a more compact appearance.

Reviewer 2: I'm probably wrong but the pill variant only changes the border-radius, so I wouldn't expect this to say it's more compact.

Author: Removed the "compact appearance" reference. Thanks

> - **Navigation:** NavExpandable now supports icon integration.

Reviewer 2: I'd probably use a regular word over the component name `NavExpandable`.I might use the [summary in the PR](https://github.com/patternfly/patternfly-react/pull/12444).

"Expandable navigation items now support an optional icon prop that displays before the title."

Author: Done! Thanks.
